backend/app/modules/user/tos/service/basic.py

```diff
@@ -0,0 +1,44 @@
+from app.extensions import db
+from app.context.manager import Context
+from app.exceptions import DuplicateConsent, InvalidSignature
+from app.infrastructure import database
+from app.modules.common.verifier import Verifier
+from app.modules.user.tos.core import verify_signature
+from app.pydantic import Model
+
+
+class BasicUserTosVerifier(Verifier):
```

Comment:
hmm, so it seems that it's misleading, because it's was not meant to suggest that it's EOA verifier. I changed it to `InitialUserTosVerifier`, maybe that would be better ;)